src/main/kotlin/com/hj/leetcode/kotlin/problem703/Solution.kt

```diff
@@ -0,0 +1,40 @@
+package com.hj.leetcode.kotlin.problem703
+
+import java.util.*
+
+/**
+ * LeetCode page: [703. Kth Largest Element in a Stream](https://leetcode.com/problems/kth-largest-element-in-a-stream/);
+ */
+class KthLargest(private val k: Int, nums: IntArray) {
+
+    private val minPq = PriorityQueue<Int>()
```

Comment:
`kLargest` may be a better name

Reply:
Resolved at https://github.com/hj-core/hj-leetcode-kotlin/pull/68/commits/c6ea08e824a7eb337d9cc7649e0ee5bb14247729